### PR TITLE
Change FileInputStream Stats readTimeUs to readTimeNs

### PR DESCRIPTION
**Pull request by jinchengchenghh**
Resolves: https://github.com/facebookincubator/velox/issues/10688

### CODE REVIEW

**Comment by jinchengchenghh** [comment]
Can you help review? Thanks! @xiaoxmeng

File: velox/common/file/FileInputStream.cpp [hunk 1]
```diff
@@ -56,9 +56,9 @@ void FileInputStream::readNextRange() {
   current_ = nullptr;
 
   int32_t readBytes{0};
-  uint64_t readTimeUs{0};
+  uint64_t readTimeNanos{0};
```
Comment (xiaoxmeng) [inline velox/common/file/FileInputStream.cpp:59]: nit: s/readTimeNanos/readTimeNs/

File: velox/common/file/FileInputStream.cpp [hunk 2]
```diff
@@ -222,9 +222,9 @@ void FileInputStream::maybeIssueReadahead() {
   VELOX_CHECK(readAheadWait_.valid());
 }
 
-void FileInputStream::updateStats(uint64_t readBytes, uint64_t readTimeUs) {
+void FileInputStream::updateStats(uint64_t readBytes, uint64_t readTimeNanos) {
```
Comment (xiaoxmeng) [inline velox/common/file/FileInputStream.cpp:225]: ditto

File: velox/common/file/FileInputStream.h [hunk 3]
```diff
@@ -63,7 +63,7 @@ class FileInputStream : public ByteInputStream {
   struct Stats {
     uint32_t numReads{0};
     uint64_t readBytes{0};
-    uint64_t readTimeUs{0};
+    uint64_t readTimeNanos{0};
```
Comment (xiaoxmeng) [inline velox/common/file/FileInputStream.h:66]: ditto

File: velox/common/file/FileInputStream.h [hunk 4]
```diff
@@ -106,7 +106,7 @@ class FileInputStream : public ByteInputStream {
     return buffers_[nextBufferIndex()].get();
   }
 
-  void updateStats(uint64_t readBytes, uint64_t readTimeUs);
+  void updateStats(uint64_t readBytes, uint64_t readTimeNanos);
```
Comment (xiaoxmeng) [inline velox/common/file/FileInputStream.h:109]: ditto

File: velox/common/file/tests/FileInputStreamTest.cpp [hunk 5]
```diff
@@ -96,7 +96,7 @@ TEST_F(FileInputStreamTest, stats) {
     ASSERT_EQ(
         byteStream->stats().readBytes,
         std::min(testData.streamSize, testData.bufferSize));
-    ASSERT_GE(byteStream->stats().readTimeUs, 0);
+    ASSERT_GE(byteStream->stats().readTimeNanos, 0);
```
Comment (xiaoxmeng) [inline velox/common/file/tests/FileInputStreamTest.cpp:99]: Can we change this to ASSERT_GT?

File: velox/common/file/tests/FileInputStreamTest.cpp [hunk 6]
```diff
@@ -110,6 +110,6 @@ TEST_F(FileInputStreamTest, stats) {
         bits::roundUp(testData.streamSize, testData.bufferSize) /
             testData.bufferSize);
     ASSERT_EQ(byteStream->stats().readBytes, testData.streamSize);
-    ASSERT_GE(byteStream->stats().readTimeUs, 0);
+    ASSERT_GE(byteStream->stats().readTimeNanos, 0);
```
Comment (xiaoxmeng) [inline velox/common/file/tests/FileInputStreamTest.cpp:113]: ditto

**Review by xiaoxmeng — APPROVED** [review]
@jinchengchenghh LGTM % nits. Thanks!

**Review by Yuhta — APPROVED** [review]

**Pull request closed without merging**